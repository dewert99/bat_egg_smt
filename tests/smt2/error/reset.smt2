(set-option :print-success true)
(set-option :produce-models false)
(check-sat)
(get-model) ; fail
(reset-assertions)
(check-sat)
(get-model) ; fail
(reset)
(check-sat)
(get-model) ; succeed