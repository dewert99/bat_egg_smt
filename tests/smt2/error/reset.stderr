The option `:produce-models` must be set to true at 4:2
(get-model) ; fail
 ^

The option `:produce-models` must be set to true at 7:2
(get-model) ; fail
 ^

